common/cli.py

```diff
@@ -97,7 +97,7 @@ def errorHandler(msg):
             return False
         okay()
 
-    test = 'Check/prepair snapshot path'
+    test = 'Check/prepare snapshot path'
```

Comment:
Do we need to take care of translations here? Or is this a debug message that never gets translated?

Reply:
Makes me a headache diving deeper in this code and code that is using this part. :smile: 

In the context of this PR I would say it is OK and we should leave it.